components/dropdown/index.jsx

```diff
@@ -0,0 +1,36 @@
+import { createElement } from "complate-stream";
+
+export default function Dropdown({ id, class: additionalClass, label, variant="secondary",
+									 active, disabled, type: typeName = "button", size }, ...children) {
+
+	let classNames = ["dropdown-toggle", "btn", "btn-" + variant];
```

Comment:
You could use `btn-${variant}` here 😸

Reply:
…on the other hand, this could also be an `a` element. Is that something we even want to support?

Reply:
Tried to use `Button` first, but then I also would have to add more attributes (data-toggle="dropdown"  id={id} aria-haspopup="true" aria-expanded="false). Wasn't sure if it is a good idea to add these individual attributes. Maybe we need a solution for arbitrary additional attributes.